README.md

```diff
@@ -110,8 +110,8 @@ Want more testing? ▶ [see the Wiki](https://github.com/qdm12/gluetun/wiki/Test
     | 🏁 `PASSWORD` | | | Your password |
     | `REGION` | | One of the [PIA regions](https://www.privateinternetaccess.com/pages/network/) | VPN server region |
     | `PIA_ENCRYPTION` | `strong` | `normal`, `strong` | Encryption preset |
-    | `PORT_FORWARDING` | `off` | `on`, `off` | Enable port forwarding on the VPN server |
-    | `PORT_FORWARDING_STATUS_FILE` | `/tmp/gluetun/forwarded_port` | Any filepath | Filepath to store the forwarded port number |
+    | `PORT_FORWARDING` | `off` | `on`, `off` | Enable port forwarding on the VPN server **for old only** |
+    | `PORT_FORWARDING_STATUS_FILE` | `/tmp/gluetun/forwarded_port` | Any filepath | Filepath to store the forwarded port number **for old only** |
 

```

Comment:
Tried the new image and it starts, but the fact port forwarding cannot be enabled for the nextgen servers makes this image unusable for me.  Is there a reason why this was disabled?  I tried the old servers and port forwarding, while it can be enabled, doesn't work for me.

Reply:
Please see [that page](https://www.privateinternetaccess.com/helpdesk/kb/articles/can-i-use-port-forwarding-without-using-the-pia-client-current-gen-only) and also eventually #177.

Basically you can't use port forwarding without their application on new generation servers. Although if you can point me to the right direction (i. e. their Github open source app?) I could hook up something eventually.

For the older servers try Romania? What region are you using?

Reply:
I was using DE Frankfurt which supports port forwarding.  When I switched back to the image I was using, without changing any config (except the new VPNSP), it worked.  I also just tried Romania as well as Switzerland, same error.  Here is the error for Romania
```
vpn            | 2020-08-27T07:57:00.288-0700	ERROR	openvpn: Get "http://209.222.18.222:2000/?client_id=fced8f781a6f214451ae2f7f115b60bae68058535d5b2d4ce43c2e9ada6eca72": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
vpn            | 2020-08-27T07:57:00.288-0700	INFO	openvpn: retrying in 30 seconds
```
Also, the _**OPENVPN_TARGET_IP**_ was not working with the value I had previously.  I removed the config and ironically got assigned the IP I was using.

Reply:
It also appears some old servers that don't have a nextgen config were removed from old.  Specifically, I saw _**CA  Vancouver**_ was missing

Reply:
What image are you using where pf works? Maybe it's a firewall issue I introduced but i doubt it.

For the removed servers, I'm using the latest zip files from PIA. I would rather avoid supporting servers they don't officially support, but I can look into it.

Reply:
As mentioned on #177 CA Vancouver (and others) now work and port forwarding works too (at least on CA Vancouver) for old servers. Thanks for the help, it was definitely useful you guys trying it out! There were some silly conflict bugs between old and new PIA servers in my code, sorry about that.

There were also a bunch of fixes I made on the master branch about things here and there, that are merged in this Docker image tag too.